### ISSUE TITLE
[MNT] Some datasets missing in list of data names

### ISSUE THREAD

**Issue by hadifawaz1999** [aeon-toolkit/aeon#1828]
### Describe the issue

tsc_datasets is missing a list for new datasets introduced in the latest tsc bake off, the non-ucr ones

### Suggest a potential alternative/fix

they should be added

**Comment by TonyBagnall** [comment]
yes I should get it all fixed and working @hadifawaz1999 its tedious work though!

**Comment by hadifawaz1999** [comment]
> yes I should get it all fixed and working @hadifawaz1999 its tedious work though!

i know it is:p

**Comment by Cyril-Meyer** [comment]
PR #1982 may help a little, at least it gave the names of the datasets.

**Comment by hadifawaz1999** [comment]
Fix in #1982
